Libraries/Linking/Linking.js

```diff
@@ -12,15 +12,11 @@
 
 const InteractionManager = require('../Interaction/InteractionManager');
 const NativeEventEmitter = require('../EventEmitter/NativeEventEmitter');
-const NativeModules = require('../BatchedBridge/NativeModules');
 const Platform = require('../Utilities/Platform');
 
 const invariant = require('invariant');
 
-const LinkingManager =
-  Platform.OS === 'android'
-    ? NativeModules.IntentAndroid
-    : NativeModules.LinkingManager;
+const LinkingManager = require('./NativeLinking').default;
```

Comment:
@cpojer, @rickhanlonii -- is it a good time to just use import?

```
import NativeLinking from './NativeLinking';
```

btw @ericlewis, let's keep the variable name `NativeLinking`

Reply:
Happy to use import, and will switch variable names.